sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/FileAsyncClient.java

```diff
@@ -796,11 +811,9 @@ private void channelCleanUp(AsynchronousFileChannel channel) {
     public Mono<Void> uploadFromFile(String uploadFilePath) {
         AsynchronousFileChannel channel = channelSetup(uploadFilePath);
         return Flux.fromIterable(sliceFile(uploadFilePath))
-            .flatMap(chunk -> {
-                return upload(FluxUtil.readFile(channel, chunk.start(), chunk.end() - chunk.start() + 1), chunk.end() - chunk.start() + 1, chunk.start())
+            .flatMap(chunk -> upload(FluxUtil.readFile(channel, chunk.start(), chunk.end() - chunk.start() + 1), chunk.end() - chunk.start() + 1, chunk.start())
```

Comment:
no need to change, but, probably a good idea to start keeping in mind the 120 max length